spec/rubocop/cop/rspec/hooks_before_examples_spec.rb

```diff
@@ -33,6 +50,14 @@
         ^^^^^^^^^^^^^^^^ Move `before` above the examples in the group.
       end
     RUBY
+
+    expect_correction(<<-RUBY)
+      RSpec.describe User do
+        before { setup }
+        include_examples('should be after let')
```

Comment:
guess we should reword this example to use `before` and not `let`. Now I see how you find that those two cops are nearly identical :)

Reply:
😆 

Fixed